pkgs/development/tools/prototool/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "prototool";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "uber";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0m6g5p3jyf0ja5j9rqmw8mhipdqxk7rcxm0plmh65bxk1n22fzjc";
+  };
+
+  modSha256 = "0l4nqb1c1lqqk70l9qwib1azavxlwghi2fv5siwrr5zw4jysz5ml";
+
+  subPackages = [ "cmd/prototool" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/uber/prototool";
+    description = "Your Swiss Army Knife for Protocol Buffers";
+    maintainers = [ maintainers.marsam ];
+    license = licenses.mit;
+    platforms = platforms.unix;
```

Comment:
```suggestion
```

buildGoModule already set up the platforms.

Reply:
Currently [windows is not supported](https://github.com/uber/prototool/issues/414#issuecomment-478081294)

Reply:
Fair enough, thank you for checking.